Linux build issues with file name case

Build configuration "RemoteControlSim" on a linux environment results in this error (File must start uppercase -> Emitter.hpp)


```
In file included from lib/HALRemoteControlSim/src/Board.h:65,
                 from lib/Service/src/DifferentialDrive.cpp:36:
.pio/libdeps/RemoteControlSim/ZumoHALWebots/src/WebotsSerialDrv.h:43:10: fatal error: webots/emitter.hpp: No such file or directory
   43 | #include <webots/emitter.hpp>
      |          ^~~~~~~~~~~~~~~~~~~~
compilation terminated.
```

--- a/src/WebotsSerialDrv.h
+++ b/src/WebotsSerialDrv.h
@@ -40,8 +40,8 @@
  * Includes
  *****************************************************************************/
 #include "Stream.h"
-#include <webots/emitter.hpp>
-#include <webots/receiver.hpp>
+#include <webots/Emitter.hpp>
+#include <webots/Receiver.hpp>
 
 /******************************************************************************
  * Macros

